apps/editing-toolkit/editing-toolkit-plugin/starter-page-templates/class-starter-page-templates.php

```diff
@@ -290,17 +290,18 @@ public function enqueue_assets() {
 	public function get_page_templates( string $locale ) {
 		$page_template_data   = get_transient( $this->get_templates_cache_key( $locale ) );
 		$override_source_site = apply_filters( 'a8c_override_patterns_source_site', false );
-		$is_assembler_v2_site = in_array( get_stylesheet(), array( 'pub/assembler', 'assembler' ), true );
+		// Enable testing v2 page patterns for Automatticians
+		$enable_testing_v2_patterns = function_exists( 'is_automattician' ) ? is_automattician() : false;
```

Comment:
Thanks. We are trying to make it easy for testers and we decided that for all Automatticians works too.

Reply:
BTW. I tried it but it's not working. The `$_GET` has only:

```
Array
(
    [_envelope] => 1
    [_gutenberg_nonce] => 231c322f96
    [_locale] => user
)
```

Because that code is triggered by the request `public-api.wordpress.com/wp/v2/sites/{ SITE }/block-patterns/patterns?_envelope=1&_gutenberg_nonce=231c322f96&_locale=user`.

Any ideas?